### PR TITLE
Fixed Homepage!

### PR DESCRIPTION
**Pull request by thealphacod3r**
Things left

- Animations to added to the homepage

- Color Scheme will be Changed if required

- Footer is to be done

### CODE REVIEW

**Comment by smaranjitghose** [comment]
add_illustrations_for_each
- Add illustrations for each of the features
this is to be present in the nav_bar

Doc2Pen and the TagLine should be in the video and the logo in the navbar and as a  favicon. Remove this

File: index.html [hunk 1]
```diff
@@ -87,5 +95,52 @@
...
+          <img id="img-logo" src="/assets/favicon/logo.png">
+        </div>
+    </div>
+
+    <div class="what">
+        <p id="what-text">What is DOC2PEN?</p>
+        <img id="what-image" src="assets/images/To_do_list.svg">
+        <p id="what-ans">DOC2PEN is a student's saviour to submit the assignments by just typing them out and getting it handwritten on the go! It simply converts image to handwritten document and and also allow us to download documents easily.</p>
+    </div>
+
```
Comment (smaranjitghose) [inline index.html:125]: Put more comments for each section

**Comment by thealphacod3r** [comment]
@smaranjitghose Ok will do

File: index.html [hunk 2]
```diff
@@ -87,5 +95,52 @@
...
         </script>
-    </body>
+
+    <!---------------------------------->
+
+
+    <!-----------Main Body--------->
+    <div id="video">
+      <video autoplay loop>
+        <source src="assets/video/homepage/doc2pen video.m4v" type="video/mp4">
```
Comment (smaranjitghose) [inline index.html:105]: m4v is more preferrable for Apple Devices and if you check the deploy at https://deploy-preview-31--doc2pen.netlify.app/, it's neither playing on a windows system or Ubuntu System

File: index.html [hunk 3]
```diff
@@ -4,54 +4,60 @@
       <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
       <link type="text/css" rel="stylesheet" href="assets/css/materialize.min.css"  media="screen,projection"/>
       <link rel="stylesheet" href="assets/css/style.css">
+      <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
```
Comment (smaranjitghose) [inline index.html:7]: We are using Montserat or Poppins for the entire page. Always keep fonts consistent throughout the project.
Please comment even the smallest things like export of fonts

File: assets/css/style.css [hunk 4]
```diff
@@ -337,3 +337,129 @@ p.demos {
...
+	margin:0px;
+	background-color: white;
+	height: 90vh;
+}
+
+#header { 
+	float: left;
+}
+
+#doc2pen {
```
Comment (smaranjitghose) [inline assets/css/style.css:362]: Make use of more meaningful names in the css files

File: assets/css/style.css [hunk 5]
```diff
@@ -337,3 +337,129 @@ p.demos {
 		transform: rotate(-360deg);
 	}
 }
+
+body {
+	font-family: 'Roboto', sans-serif;
```
Comment (smaranjitghose) [inline assets/css/style.css:342]: Change to Poppins or Montserat

File: index.html [hunk 6]
```diff
@@ -87,5 +95,52 @@
...
+      <ul id="row">
+        <li id="column">Convert Typed Document to Handwritten</li>
+        <li id="column">Custom Fonts Available</li>
+        <li id="column">Download result in various formats</li>
+        <li id="column">Open Source</li>
+      </ul>
+    </div>
+
+    <div class="contribute-block">
+      <p id="contribute-heading">Think You can <br><span id="contribute">Contribute?</span></p>
```
Comment (smaranjitghose) [inline index.html:137]: This should be in the contact page. Home page is only meant to serve as a pictorial elevator pitch

File: index.html (hunk 2 above)
Comment (thealphacod3r) [reply index.html:105]: it's playing on mine

**Pull request closed without merging**